Ayush/tech talk edits (#6)
ascend-io/ascend-community: ottos_expeditions/README.md

**Objective**: Analyze customer cohorts to understand booking behaviors and trends.

  ### 1. Overview

>   In this expedition, you'll work with customer data to categorize customers into cohorts and update reports for downstream consumers in PowerBI. To start open the `flows/1-customer_cohort_analysis` folder in the File Explorer on the left.

Reviewer: ```suggestion
  In this expedition, you'll work with customer data to categorize customers into cohorts and update reports for downstream consumers in PowerBI. To start, open the `flows/1-customer_cohort_analysis` folder in the File Explorer on the left.
```

  - Otto will provide you with the code to update the transformation component. You can copy and paste the change or let Otto update the file directly for you (you just need to confirm the changes you want him to make).
  - Once the file is updated, click **Save** to save the changes.

### 4. Build & Run the Project

As with Expedition 2, since changes have been made, we will need to rebuild the project. The steps have been restated here but are the same process:

  1. Navigate to the bottom bar of your browser. Here, you should see a couple of items such as the build status, what branch you're on, and the last commit. 
  2. Click on the build status, which should be one of the following:
      - `No build`
      - `Build successful`
      - `Build failed`
  3. On the window that pops up, click **Build Project**.
  4. Once the project finishes building, click on **guide_performance_summiting_success_rates** in the **Build Explorer** panel and run the flow to see the results.
      - In the case where the **Build Explorer** panel is not already open, click on **Show Build Panel** in the bottom right.

### 5. Congratulations!

> Amazing! You've completed Expedition 3 and on the way, made friends with Otto, the ever helpful AI! Ready for the next adventure?

Reviewer: ```suggestion
Amazing! You've completed Expedition 3 and on the way, made friends with Otto, your ever helpful AI assistant! Ready for the next adventure?
```

- Click **Save** to save the changes.

### 4. Build & Run the Project

Repeat the same process as the previous expeditions to build the project. Once the build is complete, run the **route_difficulty_success_prediction** flow to see your changes.

### 5. Congratulations!

> Fantastic job making it to the end of expedition 4! The different "views" on the way to the top sure are great, huh? Let's keep going!

Reviewer: this is amazing. 😂

### 1. Overview
In this flow, you'll work with revenue and cost data to analyze the financial health of the company. Now that you've got a good grasp on the basics of the platform, you'll be building this flow from scratch. But don't worry, we've got you covered with step-by-step instructions.

We'll be learning about the following concepts:
  1. How to create and write the code of a new Read Component
  2. How to create and write the code of a new SQL transform

>   3. How to create and make your own fully functioning Ascend Flow!

Reviewer: ```suggestion
  3. How to build and run your own fully functioning Ascend Flow!
```

  - `data/financial.csv`

### 2. Set up the flow
  - To start, you'll need to create a new folder in the `flows` directory. Let's name it `7-revenue_cost_analysis`. This folder will host all the assets for this flow.
  - Inside the `7-revenue_cost_analysis` folder, create a new folder called `components`. Ascend looks for a `components` folder in each flow directory to understand the components in the flow.

>   <!-- TODO: Don't we need to also have them create a flow definition YAML? -->

Reviewer: I think we need to resolve this. I believe the do need to create this. So, let's add this step before they create the components file? Something like:

```suggestion
- Within the `7-revenue_cost_analysis` folder, create a new yaml file names `7-revenue_cost_analysis.yaml`. This file will define the flow. In advance use cases, you can use these flow definition files to define parameters for a specific flow. For now, we'll leave this empty. 
```
